src/openseadragon.js

```diff
@@ -341,6 +341,10 @@
   * @property {Boolean} [gestureSettingsMouse.clickToZoom=true] - Zoom on click gesture
   * @property {Boolean} [gestureSettingsMouse.dblClickToZoom=false] - Zoom on double-click gesture. Note: If set to true
   *     then clickToZoom should be set to false to prevent multiple zooms.
+  * @property {Boolean} [gestureSettingsMouse.dblClickDragToZoom=false] - Zoom on dragging through
```

Comment:
This is great. We need one of these each for `gestureSettingsTouch`, `gestureSettingsPen`, and `gestureSettingsUnknown`.